### PR TITLE
ShoppingCart and Ticket dao and service implemented

### PR DESCRIPTION
**Pull request by DmGrt**

### CODE REVIEW

**Review by IrynaDim — APPROVED** [review]
i think evrything is great =) good job!

File: src/main/java/com/dmgrt/cinema/models/Ticket.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "tickets")
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne
+    private MovieSession movieSession;
```
Comment (RomanDubovskyi) [inline src/main/java/com/dmgrt/cinema/models/Ticket.java:21]: Can movieSession have only one ticket?

File: src/main/java/com/dmgrt/cinema/models/Ticket.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    @OneToOne
+    private User user;
```
Comment (RomanDubovskyi) [inline src/main/java/com/dmgrt/cinema/models/Ticket.java:23]: can user have only one ticket?

File: src/main/java/com/dmgrt/cinema/security/impl/AuthenticationServiceImpl.java [hunk 3]
```diff
@@ -29,6 +32,7 @@ public User register(String email, String password) {
         User user = new User();
         user.setEmail(email);
         user.setPassword(password);
+        shoppingCartService.registerNewShoppingCart(user);
```
Comment (RomanDubovskyi) [inline src/main/java/com/dmgrt/cinema/security/impl/AuthenticationServiceImpl.java:36]: is it logical to add cart first then user?

File: src/main/java/com/dmgrt/cinema/models/ShoppingCart.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "shopping_cats")
+public class ShoppingCart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```
Comment (RomanDubovskyi) [inline src/main/java/com/dmgrt/cinema/models/ShoppingCart.java]: is this annotation necessary with mapsId?

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**